utils/src/main/java/com/cloud/utils/ssh/SshHelper.java

```diff
@@ -184,12 +185,11 @@ public static void scpTo(String host, int port, String user, File pemKeyFile, St
 
             sess.execCommand(command);
 
-            InputStream stdout = sess.getStdout();
-            InputStream stderr = sess.getStderr();
-
-            byte[] buffer = new byte[8192];
-            StringBuffer sbResult = new StringBuffer();
+            StreamGobbler stdout = new StreamGobbler(sess.getStdout());
+            StreamGobbler stderr = new StreamGobbler(sess.getStderr());
 
+            byte[] buffer = new byte[16384];
+            StringBuffer sbResult = new StringBuffer(214748364);
```

Comment:
@Pearl1594 what's this magic number?

Reply:
@rhtyd the number represents the number of characters that corresponds to 200MB